code/modules/cargo/packs.dm

```diff
@@ -630,12 +630,12 @@
 
 /datum/supply_pack/security/armory/fire
 	name = "Incendiary Weapons Crate"
-	desc = "Burn, baby burn. Contains three incendiary grenades, three plasma canisters, and a flamethrower. Requires Armory access to open."
+	desc = "Burn, baby burn. Contains three incendiary grenades and three prebuilt flamethrowers. Requires Armory access to open."
 	cost = 4500
-	contains = list(/obj/item/flamethrower/full,
-					/obj/item/tank/internals/plasma,
-					/obj/item/tank/internals/plasma,
-					/obj/item/tank/internals/plasma,
+	contains = list(
+		/obj/item/flamethrower/full,
+					/obj/item/flamethrower/full,
```

Comment:
u didnt move the indentation of the other ones down

Reply:
fixed, i am the stupid